setup.py

```diff
@@ -34,7 +34,7 @@ def get_long_description():
     license='MIT',
     url='https://github.com/albu/albumentations',
     packages=find_packages(exclude=['tests']),
-    install_requires=['numpy>=1.11.1', 'scipy', 'opencv-python', 'imgaug'],
+    install_requires=['numpy>=1.11.1', 'scipy', 'opencv-python', 'imgaug>=0.2.5'],
```

Comment:
Could you please clarify why this change is made? Are there any interaction problems with albumentations and imgaug < 0.2.5?

Reply:
Existing tests fail with imgaug < 0.2.5 due to `AttributeError: module 'imgaug.augmenters' has no attribute 'PerspectiveTransform'` (it was added in 0.2.5 only).